src/Deck.js

```diff
@@ -0,0 +1,10 @@
+class Deck {
+  constructor(array) {
```

Comment:
I really prefer to have parameters set to not be the same words as the key for the key value pair it coresponds to. This is also a helpful reminder to pass in an array and not just a card.